Dockerfile

```diff
@@ -17,14 +18,18 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     python3-pip \
     python3-setuptools \
     software-properties-common \
+    tar \
     unzip \
     wget \
     zip \
     && rm -rf /var/lib/apt/lists/*
 
 # Install python packages
-RUN pip3 install argcomplete colorama nose2
+RUN pip3 install nose2 qmk
```

Comment:
Whats more likely to change, the qmk cli or the gcc-arm-none-eabi version? It might be worth moving the pip3 line below the whole wget thing to get better layer caching.

Reply:
Good call, I've moved this down.